model/states.go

```diff
@@ -92,44 +92,6 @@ func (b *BaseState) MarshalJSON() ([]byte, error) {
 	return cus, err
 }
 
-func (b *BaseState) UnmarshalJSON(data []byte) error {
```

Comment:
will this be unmarshaled as part of the states?

Reply:
Yes. json.Unmarshal does the work.